app/components/highcharts.js

```diff
@@ -16,7 +16,8 @@ export default class HighchartsComponent extends Component {
   @tracked isChartDrawn = false;
 
   get chartOptions() {
-    const { chartOptions, data } = this.args;
+    const { options: chartOptions, data } =
+      this.args.chart?.highchartsOptions || {};
```

Comment:
As an aside, when you want to provide a default value, I recommend using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator), which has the syntax of `??`.

The OR operator, `||`, allows the 2nd operand to take when the 1st operand is falsey. On the other hand, the `??` allows the 2nd operand only when the 1st operand is either `undefined` or `null`.

I think having limited, very specific choices makes it less likely to introduce bugs and makes your intent (that the 2nd operand, `{}`, is the default value) more clear.